src/main/scala/com/redis/sentinel/SentinelMonitor.scala

```diff
@@ -111,13 +118,17 @@ class SentinelMonitor (address: SentinelAddress, listener: SentinelListener, con
 
 trait SentinelHeartBeater extends Runnable with Log{
   private var running = false
+  private val beatingSentinel = new AtomicBoolean(true) // initialised to true so we don't unnecessarily wake up ops
```

Comment:
You mean stale cause we only periodically update that value? If you have no big objection to using AtomicBoolean, I don't see any really, I'd prefer to keep it in.